test/extended/testdata/test-deployment-test.yaml

```diff
@@ -14,7 +14,10 @@ spec:
         execNewPod:
           containerName: myapp
           command:
-          - /bin/echo
+          - /bin/sh
+          - -c
+          args:
+          - echo
```

Comment:
Add a new test instead of editing previous tests. I am not even sure which tests are using this fixture,